masonry/src/contexts.rs

```diff
@@ -6,6 +6,7 @@
 use std::time::Duration;
 
 use accesskit::{NodeBuilder, TreeUpdate};
+use kurbo::Vec2;
```

Comment:
I think it doesn't make a difference, they resolve to the same symbol. (Otherwise I think rustc would complain.)

I don't know if we have a coding style for this, but either way, we're already using both version throughout Masonry, so I'll keep the ones you pointed out as-is.

Reply:
I think it doesn't make a difference, they resolve to the same symbol. (Otherwise I think rustc would complain.)

I don't know if we have a coding style for this, but either way, we're already using both version throughout Masonry, so I'll keep the ones you pointed out as-is.